third_party/BUILD.bazel

```diff
@@ -0,0 +1,2 @@
+# SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Should this be Apache 2 as well?

Reply:
After a rewrite I changed copyright everywhere where possible to BSD - if you want to have Apache 2.0 that would also be possible